check

```diff
@@ -23,14 +23,14 @@ else
     case ${fix:0:1} in
         n|N )
             echo '❌ php-cs-fixer FAIL'
-            offer_run 'php-cs-fixer fix --config=.php-cs-fixer.php'
+            offer_run './vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php'
         ;;
         * )
             if (php-cs-fixer fix --config=.php-cs-fixer.php > /dev/null 2>/dev/null); then
                 echo '✅ php-cs-fixer OK'
             else
                 echo '❌ php-cs-fixer FAIL'
-                offer_run 'php-cs-fixer fix --config=.php-cs-fixer.php'
+                offer_run './vendor/bin/php-cs-fixer fix --config=.php-cs-fixer.php'
```

Comment:
php-cs-fixer is meant to be installed globally in local development

Reply:
@stancl I think we should run in project scope instead of global because not all of contributors install php-cs-fixer globally

Reply:
This is standard across our project so we won't be changing it here. There's no need to run php-cs-fixer locally, the CI does it for you when contributing. So it's fine if `./check` doesn't work for people who don't have php-cs-fixer installed.

I think it's more standard to use php-cs-fixer globally than in dev dependencies anyway. Part of the reason is that its dependencies can conflict with the project dependencies.

Reply:
@stancl Thanks, I will revert it

Reply:
@stancl I have removed php-cs-fixer in requirement